docker-compose.dev.yml

```diff
@@ -1,12 +1,10 @@
 version: "3.7"
 services:
   api:
+    image: "marquezproject/marquez:${TAG}"
     build: .
 
   web:
     build:
       context: ./web
       dockerfile: Dockerfile
-
```

Comment:
Oh cool! I didn’t realize adding `image` would tag the image when used with `build`.  Mind if we also add `image` to the `web` service for consistency?

```yaml
web:
  image: "marquezproject/marquez-web:${TAG}"
...
```

Reply:
In this PR, mind if we set `TAG` to `latest` when using the `--build` in [`docker/up.sh`](https://github.com/MarquezProject/marquez/blob/main/docker/up.sh)?

```
-b|'--build')
    BUILD='true'
    TAG='latest'
    ;;
```

This will avoid the image for an official release being cached with the same tag. For example, if we build with `TAG=0.17.0` locally, the official tag `0.17.0` won't be pulled if `--tag 0.17.0` was used.